snp_blending/config/app.rviz

```diff
@@ -457,6 +457,30 @@ Visualization Manager:
         Reliability Policy: Reliable
         Value: /industrial_reconstruction_mesh
       Value: true
+    - Class: tesseract_rviz/TesseractWorkbench
+      Enabled: true
+      Environment Properties:
+        Display Mode: Monitor
+        Joint State Topic: /joint_states
+        Monitor Topic: /snp_environment
+        Snapshot Topic: /tesseract_environment_snapshot
+        URDF Parameter: robot_description
+        Value: ""
+      Joint Trajectory Properties:
+        Legacy Joint Trajectory:
+          Value: true
+          topic: /joint_trajectory
+        Tesseract Joint Trajectory:
+          Joint Trajectory Topic: /tesseract/display_tesseract_trajectory
+          Value: true
+        Value: ""
+      Name: TesseractWorkbench
+      Value: true
+      tesseract::EnvMonitorJointStateTopic: /joint_states
+      tesseract::EnvMonitorMode: Monitor
+      tesseract::EnvMonitorSnapshotTopic: /snp_environment
+      tesseract::EnvMonitorTopic: /snp_environment
+      tesseract::EnvMonitorURDFDescription: robot_description
```

Comment:
There's currently a bug with tesseract_rviz in tesseract_ros2 that has been causing some crashes when a new config file is saved. Are you still able to play the trajectories without any difficulties?

Reply:
Mostly; it usually doesn't crash. I've noticed that I need to toggle the mode between URDF and monitor to get the visualization to show up. I usually change and revert the trajectory topic as well to make sure the trajectory gets saved into the widget. We should look into this on `tesseract_ros2`